pvlib/modelchain.py

```diff
@@ -17,7 +17,7 @@
                    temperature, tools)
 from pvlib.tracking import SingleAxisTracker
 import pvlib.irradiance  # avoid name conflict with full import
-from pvlib.pvsystem import _DC_MODEL_PARAMS
+from pvlib.pvsystem import _DC_MODEL_PARAMS, _unwrap_single_value
```

Comment:
Not sure this is necessary (see below) If we want to use it we should move `_unwrap_single_value` to `pvlib.tools` since it is becoming useful outsize `pvlib.pvsystem`